### ISSUE TITLE
Re-enable resource update in build before merging release-v2.1.1 into develop branch

### ISSUE THREAD

**Issue by PhotoNomad0** [unfoldingWord/translationCore#6706]
### User Story

As a [type of user], I want [some goal] so that [some reason].

### Features / Specifications

- [ ]
- [ ]
- [ ]

### Definition of Done

- [ ]
- [ ]
- [ ]

**Comment by birchamp** [comment]
@PhotoNomad0 has this been addressed already or is it blocked by another issue?

**Comment by PhotoNomad0** [comment]
already done.
